models/format_bits.go

```diff
@@ -28,7 +28,10 @@ func GetEnumBitsFormatted(value interface{}, flags Format, enum *Enum) (v Value)
 	if flags == FormatNone {
 		return
 	}
-	octets := value.([]byte)
+	octets, ok := value.([]byte)
+	if !ok {
+		return
+	}
```

Comment:
This library has an internal helper for this
```suggestion
	octets := v.Bytes()
```

Reply:
There is a `Bytes()` function for the [value type](https://github.com/sleepinggenius2/gosmi/blob/1cf1572629bc66f200fe9b4ffdfd4d2446e856a5/models/format.go#L45), but v here is an interface, not value.

Reply:
`value` is an interface, `v` is a `Value`, see: `func GetEnumBitsFormatted(value interface{}, flags Format, enum *Enum) (v Value)`

Reply:
I see now. And can understand why this was missed previously when trying to operate on value vs v.